core/node/house_keeper/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "zksync_house_keeper"
+version = "0.1.0"
+edition.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+keywords.workspace = true
+categories.workspace = true
+
+[dependencies]
+vise.workspace = true
+zksync_dal.workspace = true
+zksync_shared_metrics.workspace = true
+zksync_utils.workspace = true
+zksync_db_connection.workspace = true
+prover_dal.workspace = true
+zksync_types.workspace = true
+zksync_config.workspace = true
+
+async-trait.workspace = true
+tokio = { workspace = true, features = ["time"] }
+anyhow.workspace = true
+tracing.workspace = true
+metrics.workspace = true
```

Comment:
Would it be too much to rewrite the house keeper metrics to use `vise`? IIRC, it's essentially the only core component that still uses them. Could be done separately, of course.

Reply:
>IIRC, it's essentially the only core component that still uses them
does not looks so(photo1)
<img width="463" alt="image" src="https://github.com/matter-labs/zksync-era/assets/72273339/b191069d-7df8-44ab-bc9a-439da54f4514">


Taking into account that this is not the only one I would prefer to do this in the separate PR.